sentences/cs/_common.yaml

```
@@ -7,11 +7,60 @@ responses:
     no_device_class: "{{ area }} neobsahuje {{ device_class }}"
     no_entity: "Žádné zařízení ani entita se nejmenuje {{ entity }}"
     handle_error: "Při zpracování došlo k neočekávané chybě."
-lists: {}
+lists: 
+  color:
+    values:
+      - in: "bíl(á|ý|é|ou)"
+        out: "white"
+      - in: "čern(á|ý|é|ou)"
+        out: "black"
+      - in: "červen(á|ý|é|ou)"
+        out: "red"
+      - in: "oranžov(á|ý|é|ou)"
+        out: "orange"
+      - in: "žlut(á|ý|é|ou)"
+        out: "yellow"
+      - in: "zelen(á|ý|é|ou)"
+        out: "green"
+      - in: "modr(á|ý|é|ou)"
+        out: "blue"
+      - in: "fialov(á|ý|é|ou)"
+        out: "purple"
+      - in: "hněd(á|ý|é|ou)"
+        out: "brown"
+  brightness:
+    range:
+      type: "percentage"
+      from: 0
+      to: 100
+  temperature:
+    range:
+      type: "temperature"
+      from: 0
+      to: 100
+  temperature_unit:
+    values:
+      - "(celsius | celsia)"
+      - in: "c"
+        out: "Celsia"
+      - "(fahrenheit | fahrenheita)"
+      - in: "f"
+        out: "Fahrenheita"
 expansion_rules:
+  area: "{area}"
```

trivial expansion is not needed

---

Ok, but might be needed in future:  area: "[(na|v)] {area}"
```suggestion
  area: "[(na|v)] {area}"
```

---

Okay, like this it would make sense.